test/UnitTests/TestFramework.UnitTests/Assertions/AssertTests.IsInstanceOfTypeTests.cs

```diff
@@ -67,13 +67,15 @@ public void IsInstanceOfTypeUsingGenericType_WhenValueIsNull_Fails()
     [TestMethod]
     public void IsInstanceOfTypeUsingGenericType_OnSameInstance_DoesNotThrow()
     {
-        Assert.IsInstanceOfType<int>(5);
+        var result = Assert.IsInstanceOfType<int>(5);
```

Comment:
Nit: I would make the type explicit instead of `var`

```suggestion
        int result = Assert.IsInstanceOfType<int>(5);
```